agent/engine/common_integ_test.go

```diff
@@ -77,8 +77,8 @@ func setup(cfg *config.Config, state dockerstate.TaskEngineState, t *testing.T)
 		t.Skip("Docker not running")
 	}
 
-	clientFactory := clientfactory.NewFactory(context.Background(), dockerEndpoint)
-	sdkClientFactory := sdkclientfactory.NewFactory(context.Background(), dockerEndpoint)
+	clientFactory := clientfactory.NewFactory(context.TODO(), dockerEndpoint)
```

Comment:
Please do the same here, `context.TODO()` is essentially same as `context. Background()`: 

```
ctx, cancel := context.WithCancel(context.TODO())
defer cancel()
```